src/static/custom-button.ts

```diff
@@ -20,4 +22,23 @@ window.addEventListener("load", function () {
 
     const scheme_pan = document.getElementsByClassName('scheme-container')[0];
     scheme_pan.children[0].appendChild(auth_pan);
-});
\ No newline at end of file
+
+    fetchUserInfoWithInterval(login_button, logout_button, 2000);
```

Comment:
Let's create a constant variable with value 2000, to be more consistent

Reply:
Done! ✅